contracts/feature-tests/basic-features/src/lib.rs

```diff
@@ -878,6 +878,23 @@ pub trait BasicFeatures {
 		self.keccak256(&input)
 	}
 
+	// Not called, they currently just panic with "Not implemented yet!"
```

Comment:
If you want to add tests, create the mandos files, but do not add `#[test]`-s in Rust. This way we can run them in mandos-go and mandos-rs just ignores them. I did the same for multisig with deployAdder_then_call.scen.json, because contract deployment from contract is not yet implemented in mandos-rs.